.env.example

```diff
@@ -0,0 +1,4 @@
+PORT=3001
```

Comment:
You can go ahead and delete this file :)

Reply:
done!